pricing-page-exercise/kadir/index.html

```diff
@@ -4,9 +4,96 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML 5 Boilerplate</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.3/font/bootstrap-icons.css">
+    <title>Pricing for Bootstrap KCS</title>
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
+    <div class="container_all">
+      
+      <header class="container_header">
```

Comment:
I'd say it should be a `nav` inside a `header` as `nav` usually is used for holding content such as menus.

```
<header>
  <nav class="container_header>
  ...
  </nav>
</header>
```